### DISCUSSION TITLE
Unable to modify custom Torznab address after it's been added

### DISCUSSION THREAD

**Discussion by fuxxociety** [pymedusa/Medusa#10245] [Ideas & Feature requests]
I feel this could be considered both a bug and a feature request. Since I'm not getting an outright error, I'll file it as a feature request.

Now that I have Jackett configured and running, I decided to move my Jackett instance to new hardware - with a new hostname and a new local IP.

Now I have about 20 stale Torznab indexers in Medusa. No big deal, right? Just head over to Config > Search Providers > Configure Custom Torznab Providers, and select each Torznab provider and change the Provider Name / Site URL / API Key to the new one. Except it appears once a custom Torznab provider has been configured, the only thing that can be changed is the API Key and the search categories. I'm unable to select the Provider Name or Site URL, nor does the input box highlight the preexisting entry if I click the box and perform a Ctrl+A.

It appears I'll have to delete all of my custom Torznab providers and re-create new providers with the updated specifics, for all 20-ish providers.

**Comment by p0psicles** [comment]
Remove / adding 20 providers is like a 10 min exercise? Probably took you longer to create this issue?
